File: Português/meu aprendisado/linguagem c/modulo4/phonebook/phonebook.c
#include <stdio.h>

int main(void)
{
    FILE *file = fopen("phonebook.csv", "a");
    char *name = get_string("Name: ");
    char *number = "644-221-453";
    if (file == NULL)
    {
        return 1;
    }
    fprintf(file, "%s,%s\n", name, number);
    fclose(file);
}

File: Português/meu aprendisado/cs50/modulo6/phonebook.py
import csv
from sys import exit

with open("phonebook.csv", 'a') as file:
    if file == None:
        exit(1)
    peopel = [input("Your name: "), input("Your number: ")]
    write = csv.writer(file)
    write.writerow(peopel)
    
